apix_documentation_swagger_v3.yaml

```diff
@@ -197,8 +195,17 @@ info:
     properly construct requests in your application (and save you coding time).
 
     You can do so by using the following dynamic sections of this documentation.
+    In order for the endpoints to work properly, first you have to click on the
+    `Authorize` button (located before the endpoints, to the right) and enter
```

Comment:
Click that button OR click the "lock" icon in each request. That is like clicking the `Authorize` button

Reply:
Done :)